media/inf/shortcuts.inf

```diff
@@ -12,6 +12,7 @@ AccessibilityShortcuts=2, %ACCESSORIES%\%ACCESSIBILITY%
 EntertainmentShortcuts=2, %ACCESSORIES%\%ENTERTAINMENT%
 CommunicationsShortcuts=2, %ACCESSORIES%\%COMMUNICATIONS%
 GamesShortcuts=2, %GAMES%
+QuickLaunchShortcuts=26, Microsoft\Internet Explorer\Quick Launch
```

Comment:
Just to know, the "Quick Launch" string is not localized: is it the same whether one uses e.g. Polish, Russian, or English version of Windows?

Reply:
Yep, it's not localized between different Windows localizations.